backend/Dockerfile

```
@@ -0,0 +1,18 @@
+# Utilise une image Node officielle comme base
+FROM node:18
+
+# Dossier de travail dans le conteneur
+WORKDIR /app
+
+# Copie les fichiers nécessaires
+COPY package*.json ./
```

Tu copies tout dans tous les cas donc autant ne pas 2 copy
```suggestion
COPY . .
```

---

Ok c'est fait !